zfmt/dag.go

```diff
@@ -364,6 +364,9 @@ func (c *canonDAG) op(p dag.Op) {
 			c.space()
 			c.exprs(p.Args)
 		}
+	case *dag.Load:
+		c.next()
+		c.write("load %s", p.Pool)
```

Comment:
This was so that we have an output of 
```
from (
  pool 2NyhaM5nfRSwx8Wimz1rpT5KZWj
  pool 2NyhaM5nfRSwx8Wimz1rpT5KZWj
)
| merge ts:desc
| load dst <---- here
```
so by "handle other fields" do you mean something like:
```
c.next()
c.write("load %s, %s, %s, %s, %s", p.Pool, p.Branch, p.Author, p.Message, p.Meta)
```
so  you'd get `load pool, branch, author, message, meta` printed out

Reply:
Yes, but match the Zed syntax, so `load <pool>@<branch> author <author> message <message> meta <meta>` if all options are specified, `load <pool>` if just the pool is specified, `load <pool>@<branch>` if pool and branch are specified, etc.